packages/ckeditor5-ui/src/toolbar/toolbarview.js

```diff
@@ -884,6 +911,13 @@ class DynamicGrouping {
  * @member {Boolean} module:ui/toolbar/toolbarview~ToolbarOptions#shouldGroupWhenFull
  */
 
+/**
+ * To allow declarative toolbar breaks for floating toolbars it should be set to `true`.
```

Comment:
This is not exactly what this option is for. This should be more like:

> This option should be enabled for toolbars in absolutely positioned containers without width restrictions to to enable automatic {@link module:ui/toolbar/toolbarview~ToolbarView#items} grouping.
> 
> When this option is set to \`true\`, items will stop wrapping to the next line and together with {@link module:ui/toolbar/toolbarview~ToolbarOptions#shouldGroupWhenFull} this will allow grouping them when there is not enough space in a single row. 

\+ this options should be mentioned in `module:ui/toolbar/toolbarview~ToolbarOptions#shouldGroupWhenFull`.

Reply:
corrected